plugin/minimap.vim

```diff
@@ -78,6 +78,22 @@ if !exists('g:minimap_highlight_range')
     let g:minimap_highlight_range = 0
 endif
 
+if !exists('g:minimap_git_colors')
+    let g:minimap_git_colors = 0
+endif
+
+if !exists('g:minimap_diffadd_color')
+    let g:minimap_diffadd_color = 'diffAdded'
+endif
+
+if !exists('g:minimap_diffremove_color')
+    let g:minimap_diffremove_color = 'diffRemoved'
+endif
+
+if !exists('g:minimap_diff_color')
+    let g:minimap_diff_color = 'diffLine'
+endif
+
```

Comment:
It would be nice for these highlight groups to have some default setting themselves.  Currently the user has to set these (or at least, set diffLine, as far as I can tell), which might be a little annoying for people who just want a default, as far as I can imagine.  Perhaps they could be linked to the default highlight groups Vim provides for diffs, e.g. `DiffAdd`, `DiffDelete`; or perhaps they could just be colored green, red, and blue.

Reply:
Regarding the highlight groups, there's a little mismatch with how the minimap currently lets users choose highlights and these diff highlights.  `g:minimap_base_highlight` and `g:minimap_highlight` are by default a Vim built-in highlight group (`Normal` and `Title`).  It's not such a big deal, but it would be consistent to do the same with these diff highlight colors - link them to a Vim default, and let users link them to their own highlight group if they want.

Reply:
Yup, agreed. I didn't realize those highlight groups didn't ship with vim, I'll change them to `DiffAdd`/`DiffDelete`/`DiffChange`